src/ClipMirAudioReader.cpp

```diff
@@ -32,20 +32,23 @@ void ClipMirAudioReader::ReadFloats(
    float* buffer, long long where, size_t numFrames) const
 {
    std::fill(buffer, buffer + numFrames, 0.f);
-   AddChannel(0, buffer, where, numFrames);
+   auto& mutableMe = const_cast<ClipMirAudioReader&>(*this);
+   mutableMe.AddChannel(0, buffer, where, numFrames);
    if (mClip.GetWidth() == 1)
       return;
-   AddChannel(1, buffer, where, numFrames);
+   mutableMe.AddChannel(1, buffer, where, numFrames);
```

Comment:
So, I understand that staggered use of a pair of sample views doubles the number of sample blocks that keep caches, and avoids destruction of the earlier blocks too soon.  

The commit comment misleads?  The problem was not only where the fetching wraps around from end to start, but at other hops too?

It is appropriate to use mutable members instead of const-casting when the purpose is caching.

Reply:
The comment is correct at least in its intention - which isn't saying much. I suppose there's a lot of context to give to fully understand why this is needed, which would probably be overwhelming in a comment. And maybe I'm lazy, too.

Making the members mutable and `AddChannel` const.